Main.cc

```diff
@@ -109,6 +107,20 @@ void OnDisconnect(uWS::WebSocket<uWS::SERVER>* ws, int code, char* message, size
     }
 }
 
+void OnError(void* user) {
+    switch ((long)user) {
+        case 3:
+            cerr << "Client emitted error on connection timeout (non-SSL)" << endl;
+            break;
+        case 5:
+            cerr << "Client emitted error on connection timeout (SSL)" << endl;
+            break;
+        default:
+            cerr << "FAILURE: " << user << " should not emit error!" << endl;
+            exit(-1);
```

Comment:
does this mean that an error in one message from one user would crash the backend for all users?

Reply:
I think it does. I will remove the line`exit(-1);` to avoid such a case.